product-updates/security-manager-2-release-note.md

```diff
@@ -8,13 +8,31 @@ SysKit Security Manager is centralized __SharePoint and Office 365 security repo
 [Try SysKit Security Manager Free](https://www.syskit.com/products/security-manager/download/)
 
 __Product version:__ 2.0.0  
-__Build number:__ 590      
+__Build number:__ 599     
 __Release date:__ May 16, 2018  
 
 ## Features
-* __SharePoint Online and On-Premises Snapshots__ are now available in SysKit Security Manager! In addition to being able to manage SharePoint Online and On-Premises permissions, you can now take snapshots of all your site collections and Office 365 groups.
+* __SharePoint Online and On-Premises Snapshots__ are now available in SysKit Security Manager! In addition to being able to manage SharePoint Online and On-Premises permissions, you can now take snapshots of all your site collections and Office 365 groups. This is required in order to use more advanced __Permission Reports__
+
+* Use the __Permissions Reports__ screen to gather valuable information about structure, granted permissions, broken permissions inheritance, and more. We broke them down into four categories. For detailed information on every report see [this article](#internal/get-to-know-security-manager/permissions-reports-screen).
+
+    * __Permissions reports:__ 
+    
+      Auditing SharePoint permissions can be a complex and time-consuming task. Syskit Security Manager allows administrators to drill down and view all permissions, explore permissions history, and create site and user specific reports.
+      
+    * __User reports:__
+    
+       These reports can either give you a __summary__ or a __detailed view__ on the permissions that principal has within a particular site collection. 
+
+    * __Externally Shared:__
+
+      A bundle of reports concerning externally shared content and external users.  
```

Comment:
_bundle_ looks to me to be very far on the informal side of the dictionary. And using it like this is not very standard.
I'd rather use _bunch_ if you wanna go this route, _handful_ might be a middle ground, or stick with something more standard like _few_, _couple_.